.github/workflows/prod.yml

```
@@ -0,0 +1,47 @@
+name: Production
+
+# When this action will be executed
+on:
+  # Automatically trigger it when detected changes in repo
+  push:
+    branches: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)